src/platforms/javascript/common/profiling/index.mdx

```diff
@@ -42,14 +44,26 @@ Install our Electron SDK using either `yarn` or `npm`, the minimum version that
 
 <PlatformContent includePath="profiling/automatic-instrumentation-intro" />
 
-## Add Document-Policy: js-profiling header
+## Step 2: Add Document-Policy: js-profiling header
 
-In order for the profiler to start, the document response header has to include a `Document-Policy` header key with the `js-profiling` value.
+For the JavaScript browser profiler to start, the document response header needs to include a `Document-Policy` header key with the `js-profiling` value.
 
 <PlatformContent includePath="profiling/automatic-instrumentation-headers" />
 
-## Configure
+## Step 3: Configure the JavaScript SDK
 
 Configuration should happen as early as possible in your application's lifecycle. Once this is done, Sentry's JavaScript SDK will capture all unhandled exceptions and transactions.
 
 <PlatformContent includePath="profiling/automatic-instrumentation-setup" />
+
+## The Difference Between DevTools & Sentry's JavaScript Browser Profiler
+
+The main difference between Chrome DevTools and Sentry's JavaScript browser profiler is that DevTools is a developer tool for debugging problems in your code, whereas Sentry is a performance engineering tool for identifying performance bottlenecks in your system. Here are a few other notable differences:
```

Comment:
This sentence doesn't add a lot of clarity, because

- Sentry is also a developer tool for debugging problems in your code. 
- Chrome DevTools is also used for identifying performance bottlenecks.

```suggestion
What does Sentry's JavaScript browser profile offer that Chrome DevTools does not?
```